tests/unittest_manager.py

```diff
@@ -318,7 +318,7 @@ def test_borg(self) -> None:
         self.assertIs(built, second_built)
 
 
-class ClearCacheTest(unittest.TestCase, resources.AstroidCacheSetupMixin):
+class ClearCacheTest(unittest.TestCase):
```

Comment:
I'd like to get out of the usage of this mixin. I think it masks tests that don't clean up after each other (in this case, the problems were upstream in other tests). We can keep fuzzing to find the problems.

Reply:
Thank you for understanding the problem well enough so that we can stop fuzzing around 😄